Add image processing examples

c0ze opened the pull request:
fixes #162

vasilev commented on examples/image_processing/inplace/Dockerfile:
```diff
@@ -0,0 +1,13 @@
+FROM iron/ruby
```
Maybe use [`iron/ruby:dev`](https://github.com/iron-io/dockers/blob/master/ruby/dev/Dockerfile) to avoid separate installs of `ruby-dev`, `bundler`, etc ?

c0ze replied:
ah yes, that makes sense. Thank you !

c0ze commented:
ok, I updated the Dockerfiles. I also fixed the path at example in readme.

Btw, when I run with `fn run`, I get

```
/usr/lib/ruby/2.3.0/uri/rfc3986_parser.rb:67:in `split': bad URI(is not URI?): http://www.sourcecertain.com/img/Example.png (URI::InvalidURIError)
```

When running with a curl, there is no error, and it works perfectly. (Same URI)

Any ideas ? @vasilev @kunihiko-t  ? Thank you !

c0ze commented:
This should be ready to go now.
